### PR TITLE
Front page tweaks and improvements

### PR DESCRIPTION
**Pull request by travishathaway**
closes: #77

### CODE REVIEW

File: blog/2022-11-10-prefix-dev-package-search.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 slug: "prefix-div-package-search"
 tags: [prefix]
 authors: [baszalmstra]
+description: Announcing the prefix.dev website for package searching
 ---
 
 Hey everyone!
 
-Im very proud to annonce the launch of [prefix.dev](https://prefix.dev/), a fast conda package search website with many more features to come.
+Im very proud to announce the launch of [prefix.dev](https://prefix.dev/), a fast conda package search website with many more features to come.
```
Comment (beeankha) [inline blog/2022-11-10-prefix-dev-package-search.md:11]:
```suggestion
I'm very proud to announce the launch of [prefix.dev](https://prefix.dev/), a fast conda package search website with many more features to come.
```

**Comment by jaimergp** [comment]
new preview URL https://deploy-preview-76--conda-dot-org.netlify.app/

**Review by Virad01 — APPROVED** [review]

**Review by asmitbm — COMMENTED** [review]
Screenshot (514)
Minor nitpick, but can we have a `border-radius = 15px` for the image container here. It will look in-flow in my opinion.

**Review by asmitbm — COMMENTED** [review]
Group 24
And in the last meeting, we had a discussion regarding lot's of whitespace in the header. What if we implement a design like this until we get some content on the page? The bottom arrow will be animated, indicating users to scroll down.

**Comment by travishathaway** [comment]
@asmitbm,

I implemented the rounded corners you recommended.

I think we should leave the homepage design "as-is" for now though. It would be wiser to spend our energy and time towards coming up with something different rather than implementing a temporary fix.

Thanks for the review! ✌️

**Review by asmitbm — APPROVED** [review]
Looks good to me 🚀!

**Review by jaimergp — APPROVED** [review]

**Pull request merged**